test-lib.sh

```diff
@@ -487,7 +488,6 @@ function ct_npm_works() {
 
   if [ -f "$cid_file" ]; then
       docker stop "$(cat "$cid_file")"
-      rm "$cid_file"
```

Comment:
Why it is not removed?

Reply:
This is done in final `ct_cleanup` function.